README.md

```diff
@@ -223,19 +223,19 @@ print(gpt_4_model)
 
 
 # Model Predict
-model_prediction = Model("https://clarifai.com/anthropic/completion/models/claude-v2").predict_by_bytes(b"Write a tweet on future of AI", input_type="text")
+model_prediction = Model("https://clarifai.com/anthropic/completion/models/claude-v2").predict_by_bytes(b"Write a tweet on future of AI")
```

Comment:
love this PR. I think you should still show an example of setting input_type though as well because for a model that takes in "Any" to "Any" you have to specify (or use the protos in predict(), generate(), stream()) is maybe a better recommendation?

Reply:
> or use the protos in predict(), generate(), stream()) is maybe a better recommendation?

Correct.